examples/interactive_bluefog_helloworld.ipynb

```diff
@@ -573,6 +573,98 @@
     "plt.show()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Interrupt the hanged worker process\n",
```

Comment:
Consider change Interrupt to Restart or awake. Also change the API naming

Reply:
Refer to the previous comment in the interrupt_hanged_processes .